src/Talis/Html/Form/Validator/Container.php

```diff
@@ -56,42 +69,46 @@ private function add(\Talis\Html\Form\Element\aElement $element):void{
      * Generate the JS header of the validation
      * @return \stdClass
      */
-    private function header():\stdClass{
```

Comment:
same as above

Reply:
based on new changes, header should return string